golang/go.mod

```diff
@@ -1,10 +1,29 @@
 module github.com/kurtosis-tech/minimal-grpc-server/golang
 
-go 1.15
+go 1.18
 
 require (
+	github.com/improbable-eng/grpc-web v0.15.0
```

Comment:
can we write down why we're using this over grpc/grpc-web? Maybe I have missed some docs here but is - https://github.com/improbable-eng/grpc-web/issues/199#issuecomment-409497863 a driver?

Reply:
added a comment in the code